### DISCUSSION TITLE
Ctrl+L creates duplicated rendering of the prompt when SSH Linux VM

### DISCUSSION THREAD

**Discussion by spikewang** [ghostty-org/ghostty#2714] [Q&A] [answered]
My ENV:

Mac OS X Sequoia 15.1 + Debian 12 (AARCH64) in UTM. Ctrl+L works perfectly on Mac. When SSHing into Debian 12 (Default to `zsh`). When hitting Ctrl+L, the prompt will be constantly rendered repeatedly. However, it didn't happen once switching to `bash`, but ctrl+L won't refresh the screen.

![Image](https://github.com/user-attachments/assets/60ca9346-007d-4d26-8335-333779882807)

I got this habit to periodically hitting Ctrl+L to clean up my screen...

**Comment by jcollie** [comment, marked as answer]
Have you installed Ghostty's terminfo on the remote machine? See the README for details.

**Reply by spikewang** [reply]
Oh, I see. It needs special terminfo on Linux side. Yep, it works perfectly now. Appreciated it!

**Reply by michelboaventura** [reply]
What I usually to is to run an `export TERM=xterm-256color` on the remote host because I either can't install the terminfo or am lazy enough.

**Reply by spikewang** [reply]
yeah, usually, Linux VM is default to `xterm-256color`... Never thought about it. I assumed that...  

Since I need to SSH/MOSH into many VMs... Some of the minimal alpine VMs that don't even have `xterm-256color` built-in...

For my own VM, things are much easier...

**Reply by michelboaventura** [reply]
Maybe just use xterm as the terminal? I don't know which term would exist everywhere and would be enough for ghostty

**Reply by spikewang** [reply]
Let me try this out. I think $TERM is never really for compatibility anyways. It's just what is the commonly case like xterm that will mostly work in most...

**Comment by mitchellh** [comment]
Converted to a discussion. Issues are not a support channel and are only for verified bugs.
